src/applications/mhv-landing-page/components/LandingPage.jsx

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
+import { renderMHVDowntime } from '@department-of-veterans-affairs/mhv/exports';
+import DowntimeNotification, {
+  externalServices,
+} from '~/platform/monitoring/DowntimeNotification';
```

Comment:
I think this needs to be `@department-of-veterans-affairs/platform-monitoring/DowntimeNotification/containers/DowntimeNotification` unless this file is imported into a Cypress test